tsconfig.json

```diff
@@ -1,12 +1,18 @@
 {
     "compilerOptions": {
         "target": "ES6",
-        "module": "ES6",
+        "module": "ES2020",
+        "allowJs": true,
+        "checkJs": true,
+        "skipLibCheck": true,
+        "strict": true,
         "moduleResolution": "Node",
+        "allowSyntheticDefaultImports": true,
         "useDefineForClassFields": true,
         "jsx": "react-jsx",
         "jsxImportSource": "dom-renderer",
-        "lib": ["ES2023", "DOM"]
+        "lib": ["ES2023", "DOM"],
+        "outDir": "dist"
```

Comment:
代码不由 `tsc` 命令编译，这个选项其实是 Parcel 控制的，此处无效。

Reply:
要加的，不然 tsconfig 文件会报错，
> Cannot write file 'd:/1024/1/EasyWebApp/WebCell-mobile/workbox-config.js' because it would overwrite input file.

Reply:
> 要加的，不然 tsconfig 文件会报错，
> > Cannot write file 'd:/1024/1/EasyWebApp/WebCell-mobile/workbox-config.js' because it would overwrite input file.

知道了，你 `include` 了根目录文件。